src/query/procedure/module.cpp

```diff
@@ -832,8 +832,8 @@ bool SharedLibraryModule::Load(const std::filesystem::path &file_path) {
       return with_error(error);
     }
     for (auto &trans : module_def->transformations) {
-      const bool was_result_added = MgpTransAddFixedResult(&trans.second);
-      if (!was_result_added) {
+      const bool success = MGP_ERROR_NO_ERROR == MgpTransAddFixedResult(&trans.second);
```

Comment:
```suggestion
      const auto success = MGP_ERROR_NO_ERROR == MgpTransAddFixedResult(&trans.second);
```

Reply:
no no no plz no

Reply:
@kostasrim mentioned he explicitly asked you to replace `auto` with `bool`, so feel free to ignore my comment.